flutter-idea/testSrc/unit/io/flutter/bazel/FakeWorkspaceFactory.java

```diff
@@ -50,6 +51,9 @@ public static Pair.NonNull<MockVirtualFileSystem, Workspace> createWorkspaceAndF
     if (syncScript != null) {
       fs.file("/workspace/" + syncScript, "");
     }
+    if (toolsScript != null) {
+      fs.file("/workspace/" + toolsScript, "");
```

Comment:
Another potential Windows issue (not sure about filesystem details, it may be fine).

Reply:
This file is only used in tests so I don't think we need to worry about the file path here.